.github/workflows/ci-pkg-install.yml

```
@@ -68,10 +68,11 @@ jobs:
       fail-fast: false
       matrix:
         os: [ubuntu-22.04, macOS-12, windows-2022]
-        pkg-name: ["app", "lite", "pytorch", ""]
+        pkg-name: ["app", "lite", "pytorch", "lightning"]
```

```suggestion
        pkg-name: ["app", "lite", "pytorch", "lightning", ""]
```
default fro most users installing from source is None

---

If PACKAGE_NAME is not set, it defaults to "lightning" after the changes in this PR